src/components/TrackersManager.vue

```diff
@@ -0,0 +1,309 @@
+<script setup lang="ts">
+import { type ZodAffectType } from '@/types/zodAffect';
+import { useTrackers } from '@/composables/useTrackers';
+import { computed, toRef } from 'vue';
+
+const props = defineProps<{
+  flawId: string;
+  affects: ZodAffectType[];
+  mode: 'modal' | 'embedded';
+}>();
+
+const affects = toRef(props, 'affects');
+
+const {
+  trackerSelections,
+  trackersToFile,
+  fileTrackers,
+  toggleTrackerSelections,
+  unselectedStreams,
+  selectedStreams,
+  filterString,
+  isFilingTrackers,
+  alreadyFiledTrackers,
+  untrackableAffects,
+  isLoadingTrackers,
+} = useTrackers(props.flawId, affects);
+
+const availableAffectStreams = computed(() => {
+  const referenceAffect = props.affects;
+  return unselectedStreams?.value.filter((tracker) => {
+    return referenceAffect.some((refAffect) => {
+      return refAffect.ps_component === tracker.ps_component
+      && refAffect.ps_module === tracker.ps_module;
+    }
+    );
+  });
+});
+
+const selectedAffectStreams = computed(() => {
+  const referenceAffect = props.affects;
+  return selectedStreams?.value.filter((tracker) => {
+    return referenceAffect.some((refAffect) => {
+      return refAffect.ps_component === tracker.ps_component
+      && refAffect.ps_module === tracker.ps_module;
+    }
+    );
+  });
+});
+
+const emit = defineEmits<{
+  'affects-trackers:refresh': [];
+}>();
+
+function handleFileTrackers() {
+  fileTrackers().then(() => {
```

Comment:
What happens if the `fileTrackers` is rejected? do we need a `.catch`?

Reply:
I'd say that the catching should happen on the origin method `fileTrackers` at the `useTrackers` composable.